### ISSUE TITLE
Issue with 23rd September

### ISSUE THREAD

**Issue by davidindra** [igneus/church-calendar-api#12]
Hello,
thank you very much for your perfect Calendar API :) I would like to report one issue to you. I've found that 23. September should be "Památka sv. Pia z Pietrelciny, kněze", mandatory memorial. Can this be updated? ;)
Thanks and have a nice day!

**Comment by igneus** [comment]
Hi, thank you very much for this report. The celebration rank is fixed in igneus/calendarium-romanum@b88afcc. It won't, however, take effect in the API before a new version of the calendarium-romanum gem is released and deployed to the API server. I'm leaving this issue open until that happens.

**Comment by mpachas** [comment]
Hi! The same happens with July 29, new memorial of St. Martha, Mary and Lazarus.
I see that calendarium-romanum v 0.9.0 has been released on April 11th but the API (http://calapi.inadiutorium.cz/browse/general-en/2021/7) returns 'Saint Martha, memorial' on that date.
Thanks for all of this, it is great!!

**Comment by igneus** [comment]
Of course. I haven't updated the API (unlike the calendarium-romanum gem) for several years now.
